### ISSUE TITLE
How to add Feature Layers on Maps?

### ISSUE THREAD

**Issue by delir1um** [oeco/jeo#57]
Hi There! Please can you help me? I would like to know how to add feature layers onto my maps? I have imported KLM contours and borders as features in mapbox, but do not know how to get them to show on JEO / Infoamazonia. Thank you for your time!

**Comment by gufalei** [comment]
@delir1um these features do not show up in JEO so far, otherwise they would
shock with our feature posts. The layers should be created either via tile
mill and exported to mapbox or via carto db and added directly on JEO .

hope it helps

2014-08-05 8:21 GMT-03:00 delir1um notifications@github.com:

> Hi There! Please can you help me? I would like to know how to add feature
> layers onto my maps? I have imported KLM contours and borders as features
> in mapbox, but do not know how to get them to show on JEO / Infoamazonia.
> Thank you for your time!
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/oeco/jeo/issues/57.

## 

Gustavo Faleiros

**Comment by delir1um** [comment]
Hi @gufalei Thank you, so if I import the KLM contours on TileMill and export to MapBox, I would be able to add it to JEO as a layer in this fashion: 'example.44234fd23' ?

**Comment by delir1um** [comment]
@gufalei please excuse my ignorance, I'm not a coder :)

**Comment by gufalei** [comment]
exactly, if you using mapbox just add the id , like the one you send now,
if you are using cartodb, paste the whole API URL

2014-08-05 9:54 GMT-03:00 delir1um notifications@github.com:

> @gufalei https://github.com/gufalei please excuse my ignorance, I'm not
> a coder :)
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/oeco/jeo/issues/57#issuecomment-51192931.

## 

Gustavo Faleiros
